src/main/java/net/dv8tion/jda/core/requests/Response.java

```diff
@@ -61,36 +111,11 @@ public boolean isRateLimit()
 
     public JSONObject getObject()
     {
-        try
-        {
-            return responseText == null ? null : new JSONObject(responseText);
-        }
-        catch (JSONException ex)
-        {
-            return null;
-        }
+      return (JSONObject) object;
     }
 
     public JSONArray getArray()
     {
-        try
-        {
-            return responseText == null ? null : new JSONArray(responseText);
-        }
-        catch (JSONException ex)
-        {
-            return null;
-        }
-    }
-
-    public String getString()
-    {
-        return responseText;
-    }
-
-    public String toString()
-    {
-        return exception == null ? "HTTPResponse[" + code + ": " + responseText + ']'
-                : "HTTPException[" + exception.getMessage() + ']';
+        return (JSONArray) object;
     }
```

Comment:
Why remove `toString()`?
Before this change it would simply return `null` if there were no JSONArray now it throws ClassCastException